Code/Framework/AzCore/AzCore/Script/ScriptAdapterBus.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Huawei Technologies Co., Ltd. 2021-2021. All rights reserved.
+ * Description: AdapterEvents.
+ * Create: 2021-06-28
+ */
```

Comment:
Please see: 
https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#copyright-header-requirements
for copyright header requirements